test/e2e/lib/install.bash

```diff
@@ -47,7 +47,7 @@ function install_helm_operator_with_helm() {
     --set configureRepositories.repositories[0].url="https://kubernetes-charts.storage.googleapis.com" \
     --set configureRepositories.repositories[1].name="podinfo" \
     --set configureRepositories.repositories[1].url="https://stefanprodan.github.io/podinfo" \
-    --set helm.versions="${HELM_VERSION:-v2\,v3}" \
+    --set helm.versions="v2\,v3" \
```

Comment:
Can you, instead of changing this here, pass those values along as `HELM_VERSION="v2\,v3" install_helm_operator_with_helm` in the test where it is required? I think changing this made the other tests flaky.

Reply:
Actually I can't because it would break that same statement. The statement is running `helm upgrade` which maps to an alias based on the passed HELM_VERSION. And that one doesn't support comma separated list. It throws error like "no helm binary found"

Reply:
But yeah I can see how this could break other tests. HELM_VERSION=v2,v3 means that all tests will default to v2. Even when we’re testing v3. 

How about we set two env vars. One for HELM_CLI_VERSION and one for HELM_ENABLED_VERSIONS ?

Reply:
Or even better. Keep HELM_VERSION as is. But default the enabled versions like:

`HELM_ENABLED_VERSIONS=${HELM_ENABLED_VERSIONS:-$HELM_VERSION}`

Reply:
That alternative is fine with me.